src/ploneintranet/theme/content/templates/document_view.pt

```diff
@@ -0,0 +1,112 @@
+                                Share
+                            </a>
+                            <a class="icon-info-circle meta-data-toggle iconified">Toggle extra metadata</a>
+                            <fieldset class="pat-subform pat-autosubmit pat-inject" data-pat-inject="target: #document-content::before; url: /feedback/banner-notifications.html; source: #workflow-state-changed::element;">
+                                <label class="pat-select bare">
+                                    <select>
+                                        <option>Private</option>
+                                        <option>Public</option>
+                                        <option>Members</option>
+                                        <option>Pending</option>
+                                        <option>Logged in</option>
+                                        <option>Internal</option>
+                                        <option>External</option>
+                                    </select>
+                                </label>
+                            </fieldset>
+                            <button type="submit" class="icon-floppy">Save</button>
+
+                        </div>
+                    </div>
+                    <fieldset class="pat-collapsible closed meta-extra" data-pat-collapsible="trigger: .meta-data-toggle" id="meta-extra">
+                        <fieldset class="bar">
+                            <input type="text" class="pat-autosuggest" name="tags" placeholder="Tags" value="${python:','.join(context.subject)}" />
+                        </fieldset>
+                        <fieldset class="bar description">
+                            <textarea name="description" rows="8" title="Description" placeholder="Description">${context/description}</textarea>
+                         </fieldset>
+
+                        <!-- <fieldset class="bar versioning">
+                            <label>
+                                <input type="checkbox" name="cmfeditions_save_new_version" id="cmfeditions_save_new_version" /> Save a new version
+                            </label>
+                            <fieldset class="condensed pat-depend new-version-details" data-pat-depend="cmfeditions_save_new_version">
+                                <label>Upload a new file
+                                    <input type="file" name="file_file">
+                                </label>
+                                <label>
+                                    <textarea placeholder="Change notes" name="cmfeditions_version_comment" id="cmfeditions_version_comment" cols="80" rows="4"></textarea>
+                                </label>
+
+                                <button type="submit" name="submit" value="submit" class="submit">Save this version</button>
+                            </fieldset>
+                        </fieldset> -->
```

Comment:
the commented code was in the generated html from the theme. it will be cleaned up quite a lot while we add new features to the view - like editing :-)